index.html

```diff
@@ -10,8 +10,29 @@
 <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap" rel="stylesheet">
 </head>
 <body>
-  <h1>Welcome to CodeBank</h1>
+  <nav>
+    <div class="logoArea">
+      <a href="index.html">CodeBank</a>
+    </div>
 
+    <div class="navlinkArea">
+      <a href="#">How It Works</a>
+      <a href="#">Blog</a>
+      <a href="#">Installations</a>
+      <a href="#">Resources</a>
+      <a class="navBtn" href="#">Sign Up</a>
```

Comment:
use an unordered list for your links
<ul>
<li><a class='' >Blog</a></li>
....
</ul>

Reply:
Already made those changes. Check commit [4407586](https://github.com/obioraigboanusi/CodeBank-landing-page/pull/3/commits/4407586fa5a89d5fcf87348c6fd8b7ead774714d) .

Reply:
ok